### PR TITLE
[AAA Dashboard] - QA issues

### PR DESCRIPTION
**Pull request by lwyszomi**
Hi @emord 

I sent a fix for tickets:
https://dimagi-dev.atlassian.net/browse/QA-279
https://dimagi-dev.atlassian.net/browse/QA-281
Also, I added tests as you mentioned in one of the tickets.

Please let me know if you have any questions.

Regards,
Łukasz

### CODE REVIEW

File: custom/aaa/tests/test_utils.py [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+        result = build_location_filters(self.district.location_id, MINISTRY_MWCD)
+        self.assertDictEqual(result, expected_output)
+
+    def test_build_location_filters_for_district_MOHFW(self):
+        expected_output = {
+            'taluka_id': ALL,
+            'district_id': self.district.location_id,
+            'state_id': self.state.location_id,
+        }
+        result = build_location_filters(self.district.location_id, MINISTRY_MOHFW)
```
Comment (emord) [inline custom/aaa/tests/test_utils.py]: It'd be nice if these were grouped together with other MOHFW filters

File: custom/aaa/tests/test_utils.py [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+        self.assertDictEqual(result, expected_output)
+
+    def test_build_location_filters_for_state(self):
```
Comment (emord) [inline custom/aaa/tests/test_utils.py]: There should be separate state tests for MoH and MWCD

File: custom/aaa/tests/test_utils.py [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+from __future__ import unicode_literals
+from django.test.testcases import TestCase
+
+from corehq.apps.domain.shortcuts import create_domain
+from corehq.apps.locations.models import LocationType, SQLLocation
+from custom.aaa.const import MINISTRY_MWCD, ALL, MINISTRY_MOHFW
+from custom.aaa.utils import build_location_filters
+
+
+class TestUtils(TestCase):
```
Comment (emord) [inline custom/aaa/tests/test_utils.py]: Should also be a national test for both hierarchies where no locaiton filter is given

**Comment by lwyszomi** [comment]
Hi @emord 

I updated the tests

Regards,
Łukasz

**Review by emord — APPROVED** [review]

**Pull request merged**